tasks/core/apps.yml

```diff
@@ -2,41 +2,31 @@
 
 # Read all apps shipped by Nextcloud itself
 - name: Read shipped apps
-  command: ./occ app:list --shipped=true --output=json
+  command: ./occ app:list --shipped=true --no-warnings --output=json
   args:
     chdir: "{{ nextcloud_installation_dir }}"
   become: true
   become_user: "{{ nextcloud_file_owner }}"
-  register: nextcloud_shipped_apps
+  register: _result
   changed_when: false
 
 - name: Remove non-json text from command output
   set_fact:
-    nextcloud_shipped_apps: >-
-      {{
-        nextcloud_shipped_apps.stdout[
-          (nextcloud_shipped_apps.stdout.find('{')):
-        ]
-      }}
+    nextcloud_shipped_apps: "{{ _result.stdout | from_json }}"
```

Comment:
This task is not needed.

Reply:
They are needed, as explained above. Ansible does not parse the repsonse as JSON because it's simply a string in the `.stdout` key of the return object.

If you prefer we can use `_result.stdout | from_json` everywhere directly in the tasks but I refer to keep it as separate task for clarity and readability.

Reply:
https://github.com/nkakouros-original/ansible-role-nextcloud/pull/34#issuecomment-553592244

Reply:
I missed your previous comment. Could you change the task name? It is a bit misleading about what happens as it only parses the stdout as json.

Reply:
Done